@xen-orchestra/lite/src/helpers/format.ts

```diff
@@ -0,0 +1,22 @@
+import humanFormat from 'human-format'
+
+const safeHumanFormat = (
+  value: number,
+  opts: {
+    decimals?: number
+    prefix?: string
+    scale?: unknown
+    separator?: string
+    unit?: string
+  }
+) => {
+  try {
+    return humanFormat(value, opts)
+  } catch (error) {
+    console.error('humanFormat', value, opts, error)
+    return 'N/D'
+  }
+}
+
+export const formatSize = (bytes: number): string =>
+  bytes != null ? safeHumanFormat(bytes, { scale: 'binary', unit: 'B' }) : 'N/D'
```

Comment:
Is this code coming from somewhere? If so, can it be mutualized?

Reply:
Comming from [here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/utils.js#L207)